examples/rl/README.md

```diff
@@ -4,9 +4,11 @@ This folder contains scenarios that employ reinforcement learning. MARO's RL too
 
 ## How to Run
 
-The ``main.py`` script can be used to run the scenarios under ``examples/rl`` or any user-defined scenario that provides the necessary components (see the section below for details) . To choose a scenario, edit ``SCENARIO_PATH`` in the script to point to the desired scenario folder. You may also edit the rest of the config variables to your own preference. Note that this script runs in single-thread mode only.
+The ``main.py`` script can be used to run the scenarios under ``examples/SCENARIO_NAME/rl`` or any user-defined scenario that provides the necessary components (see the section below for details) . To choose a scenario, edit ``SCENARIO_PATH`` in the script to point to the desired scenario folder. You may also edit the rest of the config variables to your own preference. Note that this script runs in single-thread mode only.
 To run a scenario in multi-process mode on a local machine, you will need to use the CLI tool (which requires MARO [installation from the source](https://github.com/microsoft/maro#install-maro-from-pypi)). Start by creating a configuration file (.yml) that follows the template ``maro/maro/rl/workflows/config/template.yml`` to specify the scenario-independent settings. Then use the command ``maro local run [-c] path/to/your/config`` to run in containerized (with ``-c``) or non-containerized (without ``-c``) environments.
 
+The ``main_evaluation_only.py`` works similarly to ``main.py``. The difference is it only execute the evaluation workflow without trying to train policies. To run ``main_evaluation_only.py``, you only need to implemenent ``agent2policy``, ``policy_creator``, and ``env_sampler_creator`` in the scenario folder (see details below).
```

Comment:
without trying to train policies, **which means it can directly used for Non-RL policies.**